.github/TRANSLATING.md

```diff
@@ -0,0 +1,88 @@
+
+## Enabling a Language
+
+If you seek to contribute but find your language missing, please open an issue.
+
+### Maintainer Checklist for Adding New Languages
+
+- Update https://github.com/tauri-apps/tauri-docs/blob/next/locales.json
+- Update https://github.com/tauri-apps/tauri-docs/blob/next/.github/labeler.yml
+
+## Starting a Translation
+
+Before you start working on a translation, look through the [open pull requests](https://beta.tauri.app/contribute/translate-status#needs-review) to see if anyone else is already working on the page for your language.
+
+If none spotted, then now is time for you to lead the effort! So here's how to start:
+
+1. Head to [translation status page](https://beta.tauri.app/contribute/translate-status).
+2. Look for missing translation in corresponding languages.
+3. Make sure no one else has been working on the corresponding page.
+4. Hit `Create page +` which will help you automagically fork the repository and create the corresponding page. You might want to copy the contents of English page as a starting point.
+5. Translate the page following [translator guidelines](#translation-guidelines).
+6. Once your translation is done, initiate a pull request.
+
+Note that you may send send a pull request before your translation is complete: e.g., you may want to ask for help with translations or simply letting others know that you're already translating a specific page. However, your pull request will not be merged until the translation is complete.
+
+## Updating a Translation
+
+### Corrections
+
+If you notice spelling or grammar errors, typos, or opportunities for better phrasing, open a pull request with your suggested fix. If you see a problem that you aren't sure of or don't have time to fix, open an issue.
+
+### Broken Links
+
+When broken links are found, try to fix them across all translations. Ideally, only update the linked URLs, so that translation changes will definitely not be necessary.
+
+## Translation Guidelines
+
+Here are some guidelines to help with the translation.
+
+**Markdown Formatting**
+
+- [ ] Preserve the Markdown formatting. This includes headers (#, ##, etc.), lists (*, -, 1.), and inline styles like bold, italic, and code.
+- [ ] Ensure that links ([link text](URL)) remain functional and direct to the correct locations.
+- [ ] Ensure code blocks (triple backticks ```) are unchanged in terms of their syntax and content.
+- [ ] Double-check tables, making sure their structure remains intact.
+
+**Terminology**
+
+- [ ] Familiarize yourself with key Tauri terms and ensure they're translated consistently.
```

Comment:
```suggestion
- [ ] Familiarize yourself with key Tauri terms and ensure they're translated consistently. It may be helpful to look at other, already translated pages as a reference.
```